client/my-sites/customer-home/cards/features/blogging-prompt/prompts-navigation.jsx

```diff
@@ -0,0 +1,183 @@
+import { Button, Gridicon } from '@automattic/components';
+import { addQueryArgs } from '@wordpress/url';
+import classnames from 'classnames';
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import { Component } from 'react';
+import { connect } from 'react-redux';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import getEditorUrl from 'calypso/state/selectors/get-editor-url';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import './style.scss';
+
+export class PromptsNavigation extends Component {
```

Comment:
Any reason not to use a functional component here? My understanding is they are generally preferred for new components. (e.g. pbLl1t-pY-p2)

Reply:
No reason. I'm just used to using class components but I'll rewrite it to be functional.

Reply:
I've rewrote this to be functional component but let me know if it's missing anything. Thanks.